### PR TITLE
fix(console): long text should wrap in code editor

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

* Fixed an issue that long text in code editor did not wrap, so user was not able to see the whole content.
* Use `Roboto Mono` font face in editor

## Testing

- [x] Tested in social connector settings page, the json content can now be displayed properly.

### CODE REVIEW

File: packages/console/src/components/CodeEditor/index.tsx [hunk 1]
```diff
@@ -33,12 +33,20 @@ const CodeEditor = ({ language, isReadonly = false, value = '', onChange }: Prop
...
+            },
+          }}
           customStyle={{
             background: 'transparent',
             fontSize: '14px',
             margin: '0',
             padding: '0',
             borderRadius: '0',
+            wordBreak: 'break-all',
+            fontFamily: "'Roboto Mono', monospace",
```
Comment (gao-sun) [inline packages/console/src/components/CodeEditor/index.tsx:49]: why do we need to define same font family 3 times?

File: packages/console/src/components/CodeEditor/index.tsx [hunk 2]
```diff
@@ -33,12 +33,20 @@ const CodeEditor = ({ language, isReadonly = false, value = '', onChange }: Prop
           onChange={handleChange}
         />
         <SyntaxHighlighter
+          wrapLongLines
+          codeTagProps={{
+            style: {
+              fontFamily: "'Roboto Mono', monospace",
+            },
```
Comment (gao-sun) [inline packages/console/src/components/CodeEditor/index.tsx:40]: can it use `className` for less style-in-code?

Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx:40]: Unfortunately, no. The `react-syntax-highlighter` will generate a `pre` tag and a `code` element inside the `pre`. Both of them have inline styles by default, so I can only use inline styles to override it.

File: packages/console/src/components/CodeEditor/index.tsx (hunk 1 above)
Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx:49]: The `pre` tag and its inner `code` tag both have default inline styles that come out of `react-syntax-highlighter`. I have to override the font-family for each tag element in order to make them work consistently

File: packages/console/src/components/CodeEditor/index.tsx (hunk 2 above)
Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx:40]: Most of the styles on `pre` tag can be inherited into the inner `code`. However, `code` has its own default font-family style `monospace`. So I have to specificly override this one to keep `code` consistent with `pre`

Comment (gao-sun) [reply packages/console/src/components/CodeEditor/index.tsx:40]: i see. would you like to add some comments for explaining these 2 conversations in code?

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/components/CodeEditor/index.tsx (hunk 2 above)
Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx:40]: Sure, np

Comment (demonzoo) [reply packages/console/src/components/CodeEditor/index.tsx]: Added comments

**Pull request merged**